src/main/resources/com/nirima/jenkins/plugins/docker/builder/DockerBuilderPublisher/help-labels.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    If set, builds the image with <code>--labels</code> to add labels to the image built
+    See the docker <a href="https://docs.docker.com/engine/reference/commandline/build/">build command</a> for more information.
+</div>
```

Comment:
General note about this:
I've just checked the documentation regarding labels on the docker website and I'm sorry to say that it's woefully incomplete - there's no mention of the syntax at all 😞
So we can't rely on that site to tell users about this - we're going to have to explain to the user what the syntax is.
...and if we do that, it'd probably be best to have 3 sections to this content - the initial summary line, then a `<p>` ... `</p>` paragraph explaining the syntax required by this plugin (i.e. LabelName=LabelValue, one per line), and then the final "See ... for more information" line (which then wouldn't need a `<br/>` before it if it had a `</p>` before it).

We don't need a massive essay, but we do need to tell folks it's multiple entries (one per line) and what each line's syntax is.  Take a look at the `tagsString.html` file as an example.